data/json/Quests.json

```diff
@@ -74,7 +74,6 @@
         "name": "Chains of Harrow: Update 21.5.0",
         "date": "2017-08-16T20:44:07Z",
         "url": "https://forums.warframe.com/topic/832153-chains-of-harrow-update-2150/",
-        "imgUrl": "http://n8k6e2y6.ssl.hwcdn.net/uploads/da8dd0d10932c10f50a5d6ed3d8e7cc0.jpg",
```

Comment:
did patchlog drop all the image urls?

Reply:
That's the result of the old JSONs. Didn't push the new build yet. Should check if it's actually there then though

Reply:
I did a clean local build. They seem like they're there for some, including that one, specifically.